packages/integration-tests/src/entities/metadata.ts

```diff
@@ -43,12 +43,15 @@ import {
   tagConfig,
 } from "./entities";
 
+/** @ignore */
```

Comment:
RE: The `@ignore`'s in this file. I was on the fence about whether I should ignore everything in this file (which is what has been done) vs. use the typedoc `--exclude` option to skip this file.

I decided to go with the _ignore all the things_ route, since it seemed likely that would be the desired behaviour of any consumer of joist wishing to use typedoc.